scripts/client_connections/README.md

```diff
@@ -0,0 +1,23 @@
+# Test for checking, that there is limit of simultaneous client connections
+
+**Steps to reproduce:**
+ - Using scrips from environment/docker/pool start up the pool:
+    - ``$ cd environment/docker/pool``
+    - ``$ ./pool_start.sh``
+    - ``$ docker exec -it -u root node1 setup_indy_node_iptables && setup_iptables 9702 100``
+    - ``$ docker exec -it -u root node2 setup_indy_node_iptables && setup_iptables 9704 100``
+    - ``$ docker exec -it -u root node3 setup_indy_node_iptables && setup_iptables 9706 100``
+    - ``$ docker exec -it -u root node4 setup_indy_node_iptables && setup_iptables 9708 100``
+    
+ - Start another docker container, which contains script for creating N simultaneous client connection:
+    - ``docker run -itd --privileged --name indy-cli indycore bash``
+    - ``docker cp ../../scripts/client-connections/just_connect_N_times.py indy-cli:/tmp``
+    - ``docker cp node1:/var/lib/indy/sandbox/pool_transactions_genesis /tmp``
+    - ``docker cp /tmp/pool_transactions_genesis indy-cli:/tmp``
+    - ``docker exec -it -u root indy-cli python3 just_connect_N_times.py -g /tmp/pool_transactions_genesis -c 110``
+    - this script will try to create 110 simultaneous connections to pool.    
```

Comment:
Rise it from 110 to 150 simultaneous connections.

Reply:
Add a note that max number of connected clients to a pool is conn_limit * (n/(n - f)), so for this particular test settings max number of connected clients is 132.